### ISSUE TITLE
FNX3-17124 ⁃ [Bug] [a11y] Onboarding cards cannot be accessed one-by-one

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#3574]
### Steps to reproduce
1. Turn Talkback or Voice Assistant on
1. Slide through the Onboarding tour.

### Expected behavior
All the Onboarding tour should be presented.

### Actual behavior
The items selected and presented/read were: 

* the logo,
* private browsing icon,
* three-dot menu,
* the entire “Welcome to Firefox Preview” section.
The problem is, though all the cards are read, they cannot be selected individually, even when tapped on.
Buttons “Read our privacy notice” and “Start browsing” aren’t presented/read at all, but tapped they are selected and presented/read.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8)
* Fenix version: 1.0.0-rc.2 (#11700147)

![onboarding](https://user-images.githubusercontent.com/40694158/59838453-9acbdc00-9357-11e9-86eb-0f841c43cb3a.gif)

**Comment by softvision-miralobontiu** [comment]
Still reproducible with RC 2.0.0-rc.2 with Samsung Galaxy Tab S3 (Android 8).

**Comment by sblatz** [comment]
This seems to have been resolved. QA please verify :)

![accessibility on onboarding 2019-12-17 12_25_02](https://user-images.githubusercontent.com/4400286/71019416-44521700-20c8-11ea-82a1-4ae42efc2828.gif)

**Comment by AndiAJ** [comment]
Hi, I've checked this issue on the latest **Nightly Build #13520613 from 12/18** using the following devices:
• Google Pixel 3a (Android 9) 
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S9 (Android 8)
• Samsung Galaxy S7 (Android 7)

✔️ It works on:
• Google Pixel 3a (Android 9) 
• Huawei Mate 20 Lite (Android 8.1.0)

►Video
![20191218_144055](https://user-images.githubusercontent.com/51314259/71089248-185a9280-21a9-11ea-8698-d53ad1a8f239.gif)

❌ Doesn't work properly on:
• Samsung Galaxy S9 (Android 8)
• Samsung Galaxy S7 (Android 7)

► Video
![20191220_072724](https://user-images.githubusercontent.com/51314259/71089516-b3536c80-21a9-11ea-8612-fd617f32608f.gif)

The Google Pixel & Huawei are using using **Talkback**, whereas the Samsung devices are using the **Voice Asssitant**.

@sblatz - Please review and share your thoughts on this. ☺️ 
Could this be caused by the Voice assistant used by most (or all) Samsung devices?
Should the **logo** be labeled? didn't work on any of the above mentioned devices.

I'll remove the QA needed label until further notice.

**Comment by mcarare** [comment]
@AndiAJ As per  https://github.com/mozilla-mobile/fenix/issues/6993#issuecomment-564183070 the logo should not be selectable with Talkback.  I will investigate of we can do something about Voice Assistant.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **Nightly Build #13540607 from 12/20** using the following devices:
• Google Pixel 3a (Android 9) **[Talkback]**
• OnePlus A3 (Android 6.0.1) **[Talkback]**
• Samsung Galaxy S10+ (Android 9) **[Voice Assistant]**
• Samsung Galaxy S7 (Android 7) **[Voice Assistant]**
• Samsung Galaxy S8+ (Android 8)**[Voice Assistant]**

► Video from Samsung Galaxy S10+
![20191220_103345](https://user-images.githubusercontent.com/51314259/71242228-2c6fd280-2316-11ea-96e9-4d31f90ab9f1.gif)
